### ISSUE TITLE
Book Part II Exercises

### ISSUE THREAD

**Issue by AhmedGamal1496** [ageron/handson-ml2#110]
Hello, thank you for the great book. 
I just have a small issue that lots of exercises are not solved, yet and I truly understand as there are new parts. I just propose that may you add some hints in the exercises in jupyter notebooks, so we understand what we are trying to accomplish (e.g. train a DNN on cifar10 dataset and you must reach above 80% accuracy). Something like that at each point, so we try our best to reach that objective even if the problems are not fully solved. Thank you sir!

**Comment by nickcorona** [comment]
Yeah, I'm on chapter 13, could really use the solutions as I'm stuck on the first programming question.

**Comment by ageron** [comment]
Hi @AhmedGamal1496 and @nickcorona ,

Thanks for your kind words regarding my book, I'm glad you find it useful. :)

I'm gradually uploading the solutions to the coding exercises in part II, thanks for your patience. So far I've uploaded all solutions up to chapter 15, and I'm currently working on chapter 16.

Hope this helps!
